CMakeLists.txt

```diff
@@ -392,3 +396,7 @@ mark_as_advanced(BUILD_GMOCK)
 mark_as_advanced(BUILD_GTEST)
 mark_as_advanced(gmock_build_tests)
 mark_as_advanced(BUILD_WCE_TESTING)
+mark_as_advanced(USE_PSYCHROMETRICS_CACHING)
+mark_as_advanced(USE_GLYCOL_CACHING)
+mark_as_advanced(USE_PSYCH_STATS)
+mark_as_advanced(USE_PSYCH_ERRORS)
```

Comment:
It's really great having some new CMake (advanced) variables for turning compiler settings on and off.